vllm/distributed/parallel_state.py

```diff
@@ -381,6 +381,12 @@ def _all_gather_out_place(self, input_: torch.Tensor,
                               dim: int) -> torch.Tensor:
         return self.device_communicator.all_gather(input_, dim)
 
+    def all_gatherv(self,
```

Comment:
The `self.use_custom_call_op` in this file is set in a way that ensures the newly added ag and rs features cannot be enabled simply by toggling an environment variable.